main.rb

```diff
@@ -1,14 +1,58 @@
-require_relative 'classes/nameable'
-require_relative 'classes/person'
-require_relative 'classes/student'
-require_relative 'classes/teacher'
-require_relative 'classes/decorator'
-require_relative 'classes/capitalize_decorator'
-require_relative 'classes/trimmer_decorator'
-
-person = Person.new(22, 'maximilianus')
-puts person.correct_name
-puts capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-puts capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+require_relative 'app'
+
+def menu
+  system('cls')
+  system('clear')
+  puts '######################'
+  puts '# OOP SCHOOL LIBRARY #'
+  puts '######################'
```

Comment:
- [ ] @IsmailChangezi kindly consider remove this `puts '######################'` message they make your UI look bad, check the initial design given below for what it would look like without the puts message
![image](https://github.com/IsmailChangezi/oop-ruby/assets/61408860/eba60394-f950-4ab5-ae16-7742daee8e02)